crates/core/src/db/models.rs

```diff
@@ -273,6 +287,8 @@ pub struct Attribute<'a> {
     pub value: Option<Cow<'a, str>>,
     /// Attribute trait type
     pub trait_type: Option<Cow<'a, str>>,
+    /// Attribute generated id
+    pub id: Cow<'a, uuid::Uuid>,
```

Comment:
The model types can not be deisel types. Must always be rust type hence pulling in the uuid crate.